plugins/plex/integration.js

```diff
@@ -93,13 +93,19 @@ function update() {
         var albumTitle = '';
     }
 
+    var mediaArtElement = mediaInfoElement.children[
+        0].children[0].children[0].children[0].children[0].children[0]
     var songHash = getHashCode(songTitle)
 
-    if (songHash !== previousSongHash) {
-        previousSongHash = songHash
-        artUrl = "";
-        convertArtUrlToBase64();
-    }
+    if (mediaArtElement)
+        if (songHash !== previousSongHash) {
+            artUrl = "";
+            convertArtUrlToBase64();
+            if (artUrl === "" )
```

Comment:
``convertArtUrlToBase64`` is asynchronous and set the global artUrl variable later when the response to the XmlHttpRequest has been received and the content loaded by the FileReader. So resetting previousSongHash would cause many redundant request while the response of the first one has not been received.